api/src/pcapi/local_providers/__init__.py

```diff
@@ -4,7 +4,7 @@
 from pcapi.local_providers.cinema_providers.cgr.cgr_stocks import CGRStocks
 from pcapi.local_providers.titelive_thing_descriptions.titelive_thing_descriptions import TiteLiveThingDescriptions
 from pcapi.local_providers.titelive_thing_thumbs.titelive_thing_thumbs import TiteLiveThingThumbs
-from pcapi.local_providers.titelive_things.titelive_things import TiteLiveThings
+from pcapi.local_providers.titelive_things.titelive_things_v2 import TiteLiveThings
```

Comment:
TODO: see if we use a setting to toggle or if we just edit `pcapi.local_providers.titelive_things.titelive_things`